### DISCUSSION TITLE
Consulta QMP

### DISCUSSION THREAD

**Discussion by heinHertz** [dds-jv/foro#127] [Consultas de objetos] [answered]
Sobre el Ejercicio QueMePongo:
Hay un punto que dice:
Como usuarie de QuéMePongo, quiero poder recibir sugerencias de atuendos que tengan una prenda para cada categoría, aunque a futuro podrán tener más (Ej.: Una remera, un pantalón, zapatos y un gorro).
Estoy entendiendo que de una lista de Prendas filtra y me da una lista de Atuendos,cada una con cada categoria , está bien asi?

**Comment by roli93** [comment, marked as answer]
La idea es que modelen la sugerencia y que puedan filtrar las prendas. Para generar una sugerencia deberías realizar todas las combinaciones de prendas que tengan sentido. De todas formas, esto no es obligatorio para esta iteración, como aclaré en un mail hace unos minutos.

Si seguís con dudas, o si efectivamente querés ponerte a pensar cómo hacer esa combinatoria, avisame y lo charlamos :D!
